3-0-spring-mvc/3-0-1-hello-spring-mvc/src/main/java/com/bobocode/mvc/api/NoteRestController.java

```diff
@@ -0,0 +1,23 @@
+package com.bobocode.mvc.api;
+
+/**
+ * To make the controller work, you have to add some annotations:
+ * - @RestController - to let spring know that you want make a REST controller;
+ * - @RequestMapping("/api/notes") - to indicate the basic endpoint for all request methods;
+ * - @RequiredArgsConstructor (for using storage without explicit autowiring);
+ */
+public class NoteRestController {
```

Comment:
@erneman1 in exercises we usually write **Java Doc that describes what classes and methods do.** Not what students should be adding. Most probably some theory will be added to the README, and here in code we can say something like this

```
/**
 * NoteRestController is a rest controller that provides two HTTP endpoints. One allows you to add a new note, and
 * another one allows you to get all notes. The URL starts from `/api`. It follows REST conventions. This means that in order to add a new note, you need to perform HTTP POST request to `../notes` and pass note JSON in the request body. This endpoint also returns the created note in the response JSON. In order to get all notes, you need to perform HTTP GET to `../notes`. 
 */
```

This is just an example, and I didn't expect you to come up with such descriptions... This should be provided in the ticket. (I will prepare a document on how to create new exercises for the future).